src/app/store/actions/VehicleTestResultModel.actions.spec.ts

```diff
@@ -1,15 +1,37 @@
 import {
-  EVehicleTestResultModelActions, GetVehicleTestResultModel, GetVehicleTestResultModelSuccess, GetVehicleTestResultModelFailure
+  EVehicleTestResultModelActions,
+  GetVehicleTestResultModel,
+  GetVehicleTestResultModelSuccess,
+  GetVehicleTestResultModelFailure
 } from './VehicleTestResultModel.actions';
-import {TestResultModel} from '@app/models/test-result.model';
+import { TestResultModel } from '@app/models/test-result.model';
+import { TEST_MODEL_UTILS } from '../../utils/test-models.utils';
 
-const testResultModelExample = {
+const testResultModelExample: TestResultModel = TEST_MODEL_UTILS.mockTestResult({
+  testerStaffId: '21',
+  testStartTimestamp: '2019-01-16T12:24:38.027Z',
+  odometerReadingUnits: 'test',
+  testEndTimestamp: '2019-01-16T12:24:38.027Z',
+  testStatus: 'new',
+  testTypes: [],
+  vehicleClass: null,
+  testStationName: 'test',
+  vehicleId: '213',
+  noOfAxles: 2,
+  vehicleType: 'none',
+  countryOfRegistration: 'Ro',
+  preparerId: '231',
+  odometerReading: 123,
+  vehicleConfiguration: 'none',
+  testStationType: 'none',
+  reasonForCancellation: 'none',
```

Comment:
For testing an action, we dont need to know this much. This will reduce the file complexity if we only provide few key values